### DISCUSSION TITLE
Filter on content types for a read only user

### DISCUSSION THREAD

**Discussion by neblimokhtar** [OrchardCMS/OrchardCore#15969] [Q&A] [answered]
Hello,
I'm trying to customise a read only role so a user with this role can only preview the content. For that i need to display all content types on the filter "by content type" in the content page. The only way i found it to give this role the "Edit content for others" permission but in this case he can click on save as draft button .
Is there any other solution for this situation without giving the permession to click on the save as draft button.
Thanks

**Comment by sebastienros** [comment, marked as answer]
There are two kinds of previews:
- Previewing a draft, with the View link that is in the Content Items list
- Live Preview which will let users edit content and see it change live

I assume you can already use the first option without Edit permissions. And the second option doesn't add anything from the first one, if you don't have edit permissions.
